### DISCUSSION TITLE
Connect 2 parent nodes to a child node

### DISCUSSION THREAD

**Discussion by ghost** [freeplane/freeplane#577] [Help]
Pretty much something like what is seen in the image. This is important to me because I read a lot of research papers and I want to organize the findings of each paper such that at a glance I can see what 2 different methods provide converging evidence (the methods being parent node) for an idea (a child node). So far it seems like this program can only create a strictly linear diagram. Is there a way to create a structure like in the image?
![image](https://user-images.githubusercontent.com/71096974/185718439-fc6ec470-2fff-425f-85c8-744e372fec30.png)

**Comment by euu2021** [comment]
You can use `Insert->New node->New summary node (selected nodes)` :
![image](https://user-images.githubusercontent.com/77707706/185720726-e0900724-ca51-4b77-8c1e-0d6557d5027b.png)

But, for your situation, it won't be very useful. I recommend that you use connectors. Please, see the comments at #566 .
